src/sentry/snuba/metrics.py

```diff
@@ -866,10 +846,9 @@ def _extract_data(self, entity, data, groups):
         for op in ops:
             key = f"{op}({metric_name})"
 
-            field = _OP_TO_FIELD[entity][op].snuba_alias
-            value = data[field]
-            if field == "percentiles":
-                value = value[Percentile[op].value]
+            value = data[op]
+            if op in _OPERATIONS_PERCENTILES:
+                value = value[0]
```

Comment:
This would not be necessary if we could query `quantile(0.5)(value)` instead of `quantiles(0.5)(value)`, will follow up on that in the new year.

Reply:
Can you elaborate on this?